gpep517/__main__.py

```diff
@@ -84,12 +85,17 @@ def override_writestr(self, zinfo_or_arcname, data,
 
 
 @contextlib.contextmanager
-def patch_sysconfig(sysroot: str):
-    stdlib_path = Path(sysconfig.get_path("stdlib"))
-    if not stdlib_path.is_absolute():
+def patch_sysconfig(sysroot: Path,
+                    prefix_from: Optional[str],
+                    prefix_to: Optional[str]):
+    stdlib_path = os.path.normpath(sysconfig.get_path("stdlib"))
```

Comment:
You can't call `normpath()`. Even though `../` are unlikely here, that function is just broken by design. Also, switching back from `Path` to strings feels like a regression.

Reply:
Fixed.